tidb-lightning: clarify an ambiguous description for `sorted-kv-dir` (#7962)
pingcap/docs: migrate-aurora-to-tidb.md

# "tidb": The "tidb" backend is recommended to import data less than 1 TiB. During the import, the target TiDB cluster can provide service normally.
backend = "local"

> # Set the temporary storage directory for the sorted Key-Value files. The directory must be empty, and the storage space must be large enough to hold the dataset to be imported. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage, which can use I/O exclusively.

Author: ```suggestion
# Set the temporary storage directory for the sorted Key-Value files. The directory must be empty, and the storage space must be greater than the size of the dataset to be imported. For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage, which can use I/O exclusively.
```
